.env.template

```
@@ -16,7 +16,7 @@
 STRUCTURED_OUTPUT_FRAMEWORK="instructor"
 
 LLM_API_KEY="your_api_key"
-LLM_MODEL="openai/gpt-4o-mini"
+LLM_MODEL="openai/gpt-5-mini"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)